microsetta_public_api/server.py

```diff
@@ -81,7 +81,10 @@ def build_app():
 
     app_file = resource_filename('microsetta_public_api.api',
                                  'microsetta_public_api.yml')
-    app.add_api(app_file, validate_responses=True)
+
+    # validate_responses needs to be False to support sending binary
+    # files it seems, see https://github.com/zalando/connexion/issues/401
+    app.add_api(app_file, validate_responses=False)
```

Comment:
If we grab the `validate_responses` value from the configuration with a default to true, we can allow the server to send images without breaking current tests, modifying the configuration files.

Reply:
it looks like the test failures are asserting a 500? we shouldn't produce those? 

before changes here, i'm going to pull down to staging so we can see if this is even something we want to keep

Reply:
The tests were meant to check that validation was not allowing certain kinds of outputs. Not the most meaningful test to be honest, didn't exactly understand that a year ago when I wrote them.

Reply:
okay sounds good, can pull from config